hledger.journal

```diff
@@ -29,54 +29,65 @@ commodity ILS:BOJ
 ; what's the exact state of the main banks.
 ; For this reason we register them as `assets:banks-ancillary`.
 
+
+;; Cash flow starts here.
+;;
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
@AronNovak can we also have a line break between the `;;;;;;...` and the next line

Reply:
Sure, just pushed that.

Reply:
Thanks - please push to this repo

Reply:
I did, it was just denied without my notice :man_facepalming:

Reply:
Now it is really pushed.